src/backend/database_models/agent.py

```diff
@@ -33,6 +33,19 @@ class AgentDeploymentModelAssociation(Base):
     )
 
 
+class AgentToolAssociation(Base):
+    __tablename__ = "agent_tool"
```

Comment:
It would be more readable I think if the class name and table name are similar.

maybe table name to be `agent_tool_associations`? or change the class name to `agent_tool`

Reply:
ignore this nit ^, read through again and i dont think its a big deal. 😅, potentially worse with other naming

Reply:
Done

Reply:
I think it's better to rename it as you suggested and I did it